<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeSetting</class>
 <widget class="QDialog" name="UpgradeSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>升级设置</string>
  </property>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QComboBox" name="upgradeMode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>76</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>升级设置</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>251</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelDelayTime">
        <property name="text">
         <string>Fbc重启延时：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDelayTime">
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>秒</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fbc反馈延时：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditAckDelay">
        <property name="text">
         <string>1200</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>毫秒</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
